hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2663,10 +2663,8 @@
       tests: ["chip_sw_aes_smoketest",
               "chip_sw_aon_timer_smoketest",
               "chip_sw_clkmgr_smoketest",
-              // TODO(lowrisc/opentitan#7505): Debug CSRNG generate bits mismatch.
-              // "chip_sw_csrng_smoketest",
-              // TODO(lowrisc/opentitan#10092): Remove dependency on uncontrolled environment.
-              // "chip_sw_entropy_src_smoketest",
```

Comment:
@martin-lueker, I enabled these tests to be run in nightlies. IMO its ok to leave them in a failing state. Let me know if you are cool with this change.

Reply:
Hi Sri, 
   It's fine from my perspective.  Are these even implemented yet though?
-Martin

Reply:
Oh this is just smoketest.  Never mind.